### DISCUSSION TITLE
Add ability to follow organizations like a user

### DISCUSSION THREAD

**Discussion by simonds** [community/community#4585] [Other Feature Feedback, Questions, & Ideas] [answered]
Bringing over an issue originally posted on [isaacs](https://github.com/isaacs)/[github](https://github.com/isaacs/github) in 2013 by @junosuarez:

[Add ability to follow organizations like a user](https://github.com/isaacs/github/issues/50)

_I want to be able to see when a new repository is created in an organization that I follow, or when members are (publically [sic]) added to an organization, similar to the timeline view of new repos created by users I am following._

**Comment by khendri** [comment]
D

Pada tanggal Sab, 10 Jul 2021 10.28, Mark Simonds ***@***.***>
menulis:

**Comment by ErikSRoth** [comment]
This would be a huge asset. I am tired of having to star 20-30 repos from the same org. It is also annoying to have to keep checking on the org as a whole to see what is new. It should not be complicated to add a follow button to orgs like we have for individuals.  +50

**Comment by JARVIS-AI** [comment]
There was a issue 
And this is a long time most wanted feature
My bookmark is overwhelming For organizations

**Comment by ghost** [comment]
I can't beliave this was not implemented yet. Or was it removed at some point?

**Comment by timmyjose** [comment]
This is the Github equivalent of Trello not offering any vertical or mixed-mode layout for lists, in the browser. It's amazing that this has not been implemented yet indeed.

**Reply by amalija-ramljak** [reply]
It used to exist, but was removed long ago

**Comment by yardenac** [comment]
If this issue was a person, it'd be in 3rd grade now.

**Comment by fedemengo** [comment]
How can we start a productive discussion about this feature? Honestly it seems something quite straightforward to implement but we haven't had any feedback since the issue was originally opened 8 years ago

**Comment by bdresser** [comment]
Hi folks! This feature is in active development and will likely launch later this Spring. Thank you for your patience!

Just to make sure we're on the right track: what events from Organizations are you interested in staying on top of? All suggestions welcome :) 

cc @lukehefson

**Reply by Zenoo** [reply]
For me, following an organization should be equivalent to watching every repo of this organization.

**Reply by junosuarez** [reply]
woohoo! patience pays off :)

**Reply by lukehefson** [reply]
@Zenoo Can we ask _why_ you're interested in bulk-<kbd>Watch</kbd>ing all notifications from all repositories in a specific organisation? Could you describe a use case?

**Reply by Zenoo** [reply]
@lukehefson I'm following the [MUI](https://github.com/mui) library pretty closely, but they release some features as different repos.  
If they decide to add another feature as a whole new repo, I wouldn't know about it unless I can follow the organization and get notified directly in my Github feed.

**Reply by lukehefson** [reply]
Ah ha! Thanks @Zenoo. This is really helpful to know and I and appreciate that it's a valid annoyance with the way our current Notifications system is configured (you can currently only <kbd>Watch</kbd> to subscribe to alerts on a per-repo basis).

I would say that I see some difference between this flaw with [Notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) and what "Follow" means on GitHub. Currently, if you click <kbd>Follow</kbd> for a user– we take that as more of a social signal that you may be interested in seeing the most relevant news from someone. We then show that news in the activity feed (as opposed to explicitly signing up for **all** alerts from the person/org you follow).

**Reply by Zenoo** [reply]
I see, maybe an option to follow either the most relevant alerts **or** all alerts from a person/org would solve this issue then?  
Just like you can choose to watch all activity from a repo or specific events instead.

**Reply by svengreb** [reply]
Basically I'd like to see an organization in a way like a person's account when it comes to the events. I don't know if adding support for organizations to have stars (and the awesome new lists) will be done too, but that's what I like to see in my timeline. I follow person accounts because I like to see which repositories they add to their star(lists) and who they follow to get inspiration and “passively spread the word” about great projects and members.
Also having an (foldable) card for published releases of organization repositories is a nice way to get an overview if you don't want to explicitly go into hundreds of repositories and enable notifications explicitly.

In my case I'm also absolutely exited about this feature because I'd like to convert an person account to an organization because it was always used to represent a kind of open source brand, but the fact that I will loose all my thousands of followers and star(lists) is a total blocker to this day. Allowing users to follow an organization (and maybe enable stars for organization at the same time) will help me to help my followers to keep track of my work through their activity feed.

**Reply by lpd-au** [reply]
New repo added by an organisation or existing repo archived/hidden/deleted.

**Comment by simonds** [comment]
New repo added by an Organization. Filtering on languages and topics for new repos would be nice as well.

**Comment by davidski** [comment]
A variation on this use case, there are organizations that do work that is broadly interesting to me (the user), for example the ropensci and rlib organizations. When there are new repositories created in these organizations, I would like to be notified of their creation so that I can learn if this new project is of interest/relevance to me. I may not choose to watch all the activity in the new repos by default, and a feed of new repos along with prompting to watch/not watch (at a customizable level [releases only, etc.]) would be sufficient for this use case.

Really excited to see this show up on the road map!

**Comment by leslieam** [comment]
This would be very helpful.

**Comment by nicholas-alonzo** [comment, marked as answer]
You can now follow organizations! Just navigate to the organization page and there'll be a "Follow" button on the right! Here's the official

- [Follow organizations](https://docs.github.com/en/get-started/exploring-projects-on-github/following-organizations#following-an-organization-on-github) in the docs
- [Changelog blog post](https://github.blog/changelog/2022-03-23-following-organizations/)

**Reply by leslieam** [reply]
Thank you very much.
It really works.
Amazing.

**Reply by lpd-au** [reply]
What events does it notify you for?

**Reply by nicholas-alonzo** [reply]
If you visit an organization page you'll get a one-time notification and it tells you what events you'll be notified for (I think there were 2 events), but I already forgot! I'm sure GitHub will make a blurb about it soon!

**Reply by nicholas-alonzo** [reply]
Here's the [Follow organizations](https://docs.github.com/en/get-started/exploring-projects-on-github/following-organizations#following-an-organization-on-github) in the docs

**Reply by leslieam** [reply]
Thanks

**Reply by kmk3** [reply]
Related changelog blog post:

<https://github.blog/changelog/2022-03-23-following-organizations/>

**Reply by dipree** [reply]
@nicholas-alonzo can you update your initial post with a link to changelog and docs please, thanks.
